CrowdStrike/glide-core
# More dark mode updates

## 🚀 Description

- Include `data-viz` colors from Figma as requested.
- Updated the Tree Item `selected` hover state background color as requested.
- Revert the [border changes](https://github.com/CrowdStrike/glide-core/pull/461/files#diff-7cd871b370c0a1c3e7bccbeb64f20418992aced3f16f2eb8a30a11f2c40b5e09L155) in Dropdown and go back to the `background-color` adjustment instead after things went through another design review with Mary and Joe.
- Updated `--glide-core-shadow-sm` to work in both light and dark modes.

## 📋 Checklist



- I have followed the [Contributing Guidelines](https://github.com/crowdstrike/glide-core/blob/main/CONTRIBUTING.md).
- I have added tests to cover new or updated functionality.
- I have added or updated Storybook stories.
- I have [localized](https://github.com/CrowdStrike/glide-core/blob/main/CONTRIBUTING.md#translations-and-static-strings) new strings.
- I have followed the [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/) or met with the Accessibility Team.
- I have included a [changeset](https://github.com/CrowdStrike/glide-core/blob/main/CONTRIBUTING.md#versioning-a-package).
- I have scheduled a design review.

## 🔬 How to Test

- Go to [Tree](https://glide-core.crowdstrike-ux.workers.dev/more-dark-mode-updates?path=/story/tree--tree&globals=theme:dark) in both light and dark modes and make sure Tree isn't busted
- Verify Dropdown's latest [design](https://glide-core.crowdstrike-ux.workers.dev/more-dark-mode-updates?path=/story/dropdown--dropdown&args=add-button-label:Add+New;multiple:!true;select-all:!true&globals=theme:dark) for select all and add new (actually reverting back to the "old" one)


## 📸 Images/Videos of Functionality

| Before  | After |
| ------- | ----- |
|  <img width="515" alt="Screenshot 2024-11-22 at 3 21 18 PM" src="https://github.com/user-attachments/assets/b1b38300-3159-4796-840c-8c90c4f5a300">  | ![Screenshot 2024-11-21 at 7 47 38 PM](https://github.com/user-attachments/assets/3dc12e9b-8c89-4a26-9cea-ed779aab8fb4) |
| <img width="517" alt="Screenshot 2024-11-22 at 3 23 25 PM" src="https://github.com/user-attachments/assets/93d84dd4-b794-4d44-ae13-58a6bea0b157"> | <img width="518" alt="Screenshot 2024-11-22 at 3 22 42 PM" src="https://github.com/user-attachments/assets/8212a15f-3ba2-4649-98b6-1d6851ee37c4"> |
